packages/contracts-bedrock/test/L1/SuperchainConfig.t.sol

```diff
@@ -332,27 +241,44 @@ contract SuperchainConfig_DependencySet_Test is CommonTest {
 
     EnumerableSet.UintSet internal chainIds;
 
-    /// @notice Tests that the dependency set returns properly the dependencies added.
-    function test_dependencySet_succeeds(uint256[] calldata _chainIdsArray) public {
-        SuperchainConfigForTest superchainConfigForTest = new SuperchainConfigForTest(address(sharedLockbox));
+    function _addDependencies(uint256[] calldata _chainIdsArray) internal {
+        vm.assume(_chainIdsArray.length <= type(uint8).max);
 
         // Ensure there are no repeated values on the input array
         for (uint256 i; i < _chainIdsArray.length; i++) {
+            vm.assume(_chainIdsArray[i] != block.chainid);
```

Comment:
```suggestion
            if (_chainIdsArray[i] != block.chainid) chainIds.add(_chainIdsArray[i]);
```
I think that adding an `assume` here is a bit to overkill and will cause to discard and re-run this test an unnecessary amount of times. We could instead just decide to skip the chainId if its equal to chainId. I dont think we are affecting the fuzz testing here since we are steal keeping with the "RNG"
What do you guys think?